shielder/contract/drink_tests/utils/psp22.rs

```diff
@@ -0,0 +1,74 @@
+use drink::{
+    runtime::MinimalRuntime,
+    session::{Session, NO_ENDOWMENT, NO_SALT},
+    AccountId32, ContractBundle,
+};
+
+pub fn deploy_test_token(
+    session: &mut Session<MinimalRuntime>,
+    supply: u128,
+) -> Result<AccountId32, Box<dyn std::error::Error>> {
+    let formatted_supply: String = format!("{}", supply);
+
+    let psp22_bundle =
+        ContractBundle::load(std::path::Path::new("../PSP22/target/ink/psp22.contract"))?;
+
+    let res = session.deploy_bundle(
+        psp22_bundle,
+        "new",
+        &[
+            formatted_supply.as_str(),
+            "Some(\"TST\")",
+            "Some(\"TST\")",
+            "9",
+        ],
+        NO_SALT,
+        NO_ENDOWMENT,
+    )?;
+    Ok(res)
+}
+
+pub fn get_psp22_balance(
+    session: &mut Session<MinimalRuntime>,
+    token: AccountId32,
+    address: AccountId32,
+) -> Result<u128, Box<dyn std::error::Error>> {
```

Comment:
I suggest using `anyhow::Result<T>` all across the tests

Reply:
will try to add it later, don't get how to easily integrate anyhow, having `Box`ed `std::error:Error`.

Reply:
with `anyhow` you don't box anything

Reply:
added anyhow and replaced declarations to Result<T>